sleap/gui/overlays/tracks.py

```diff
@@ -48,8 +48,8 @@ def __attrs_post_init__(self):
     @classmethod
     def get_length_options(cls):
         if prefs["trail length"] != 0:
```

Comment:
is this conditional the correct thing to check for? won't it always be nonzero if we are using trails?